src/Parallel/GlobalCache.hpp

```diff
@@ -467,6 +472,13 @@ class GlobalCache : public CBase_GlobalCache<Metavariables> {
   /// framework. Trying to do either of these will result in an ERROR.
   void compute_size_for_memory_monitor(const double time);
 
+  /// Entry method that will set the value of the Parallel::Tags::ResourceInfo
+  /// tag to the value passed in (if the tag exists in the GlobalCache)
+  ///
+  /// This is only meant to be called once.
```

Comment:
If it should only be called once, should there be a member variable that we set once this is called to avoid multiple calls?